tools/swift-plugin-server/CMakeLists.txt

```diff
@@ -23,8 +46,50 @@ if (SWIFT_BUILD_SWIFT_SYNTAX)
       SwiftSyntaxMacroExpansion
       SwiftCompilerPluginMessageHandling
       swiftLLVMJSON
+      SwiftPluginServerSupport
   )
   target_include_directories(swift-plugin-server PRIVATE
     Sources/CSwiftPluginServer/include
   )
+
+  set(SWIFT_WASM_ALLOW_JSC ON CACHE BOOL "Use JavaScriptCore Wasm runtime if possible" FORCE)
```

Comment:
Wanted to leave this in for the initial review to show what the JSC runtime would look like, but I can remove this before we do a final pass since Doug mentioned that it's better to stick to a uniform implementation to begin with, for simplicity. I think it's definitely worth revisiting this in a follow-up because the JSC runtime is a lot faster than WasmKit when it can be used (500ms vs 1100ms in some cases I've tested.)

Reply:
Split out the JSC stuff into a separate branch, happy to make a followup PR when we're ready for JSC: https://github.com/kabiroberai/swift/compare/kabir/wasm-plugins...kabir/jsc-wasm